backend/Controllers/SightingController.cs

```diff
@@ -3,16 +3,26 @@
 using WhaleSpotting.Models.Response;
 using WhaleSpotting.Models.Database;
 
+
 namespace WhaleSpotting.Controllers {
     [ApiController]
     [Route("/sightings")]
     public class SightingController : ControllerBase {
         private readonly ISightingService _sightings;
 
-        public SightingController(ISightingService sightings)
-        {
+        public SightingController
+        (
+            ISightingService sightings
+        ) {
             _sightings = sightings;
         }
+        
+        [HttpGet] 
+        public ActionResult<ListResponse<Sighting>>GetApprovedSightings()
```

Comment:
Space between return type and method name
```suggestion
        public ActionResult<ListResponse<Sighting>> GetApprovedSightings()
```